public/grpc-client.js

```diff
@@ -93,6 +96,22 @@ module.exports.init = async function({
     });
   });
 
+  ipcMain.on('lndAtplInit', async event => {
+    credentials = await getCredentials(lndSettingsDir);
+    const macaroonCreds = getMacaroonCreds(lndSettingsDir, network);
+    credentials = grpc.credentials.combineChannelCredentials(
+      credentials,
+      macaroonCreds
+    );
```

Comment:
Same here ... have you tried reusing the credentials object combined in the lndEvent?

Reply:
hm true, I thought there was potential that autopilot would attempt to use the credentials before they're set properly [here](https://github.com/lightninglabs/lightning-app/blob/master/public/grpc-client.js#L86) but can't seem to repro that, so fixed :woman_shrugging: